cpp/src/fil/fil.cu

```diff
@@ -492,6 +569,31 @@ void init_sparse(const cumlHandle& h, forest_t* pf, const int* trees,
   *pf = f;
 }
 
+void from_multi_treelites(const cumlHandle& handle, forest_t* pforest,
```

Comment:
This contains a lot of the same code as `from_treelite()`.

Reply:
I can update these functions to check if the vector of ModelHandles is empty or not and then based on that call the required function. This will help reduce the duplicated code.